### PR TITLE
Add playbook for elife-xpub specific database restore

### PR DESCRIPTION
**Pull request by giorgiosironi**
For https://github.com/elifesciences/elife-xpub/issues/1375

### CODE REVIEW

File: docs/deploying/database.md [hunk 1]
```diff
@@ -31,3 +31,88 @@ Migrations are [run automatically](https://github.com/elifesciences/elife-xpub-f
 Migrations have to be backward compatible as every deploy has a critical section where the old application code is still running after migrations have been applied.
 
 There is no automated rollback of migrations.
+
+## Backup and restore
+
+RDS snapshots are taken daily and [retained for 4 weeks in `prod`](https://github.com/elifesciences/builder/blob/master/projects/elife.yaml#L1970).
+
+The scripts `dump-database.sh` and `restore-database.sh`, on PATH in the servers, can respectively be used to perform a new dump from any Postgres host and to restore that dump.
```
Comment (diversemix) [inline docs/deploying/database.md]: Where are these scripts? Would be good to add their location here.

File: docs/deploying/database.md [hunk 2]
```diff
@@ -31,3 +31,88 @@ Migrations are [run automatically](https://github.com/elifesciences/elife-xpub-f
...
+
+The `PGHOST` being used here is the RDS node that has just been created (2nd argument).
+
+#### 6. Restore the dump on the existing node
+
+```
+ ./restore-database.sh /ext/tmp/dump-afternoon elife-xpub-staging.cxyopn44uqbl.us-east-1.rds.amazonaws.com
+```
+
+The `PGHOST` being used here is the existing RDS node for this environment (2nd argument).
```
Comment (diversemix) [inline docs/deploying/database.md:118]: Do we need instructions for removing the "new node" ... which I guess was only done to perform the SQL extraction?

Comment (giorgiosironi) [reply docs/deploying/database.md:118]: Assuming this is done in an emergency, there's no harm in leaving it around, an administrator can remove it later on; didn't want to make this guide even longer.

File: docs/deploying/database.md (hunk 1 above)
Comment (giorgiosironi) [reply docs/deploying/database.md]: they are on the `PATH` so they should be accessible wherever you are, but I'll reword

Comment (giorgiosironi) [reply docs/deploying/database.md]: (they are in `/usr/local/bin` but you shouldn't have to know that to use them)

**Review by Cooryd — APPROVED** [review]

**Pull request merged**